### PR TITLE
[LPT] Extend OperationPrecisionRestriction to support output ports

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *Extend OperationPrecisionRestriction to support output ports*

### Tickets:
 - *73724*

### CODE REVIEW

File: src/tests/functional/inference_engine/lp_transformations/operation_precision_restriction_test.cpp [hunk 1]
```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2018-2022 Intel Corporation
```
Comment (vzinovie) [inline src/tests/functional/inference_engine/lp_transformations/operation_precision_restriction_test.cpp:1]: minor:
```suggestion
// Copyright (C) 2022 Intel Corporation
```

**Review by vzinovie — APPROVED** [review]
LGTM

File: src/tests/functional/inference_engine/lp_transformations/operation_precision_restriction_test.cpp (hunk 1 above)
Comment (eshoguli) [reply src/tests/functional/inference_engine/lp_transformations/operation_precision_restriction_test.cpp]: done, thanks!

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**